initial_pose_x: 21.431538
initial_pose_y: 9.158023
initial_pose_a: 1.606487

